spec/lib/repo/git_revision_spec.rb

```diff
@@ -3,6 +3,52 @@
 describe Repository::GitRevision do
   context 'with a git repo' do
 
+    before(:context) do
+
+      # Make sure gitolite-admin repo is cloned in test environment
+      ga_repo = Gitolite::GitoliteAdmin.new("#{::Rails.root.to_s}/data/test/repos/gitolite-admin", GITOLITE_SETTINGS)
+
+      # Bring the repo up to date
+      ga_repo.reload!
+
+      # Grab the gitolite admin repo config
+      conf = ga_repo.config
+
+      # Remove test repo from Gitolite conf
+      conf.rm_repo("test_repo")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.